internal/config/main.go

```diff
@@ -21,6 +24,10 @@ type config struct {
 	types.Copuser
 	comfig.Listenerer
 	getter kv.Getter
+
+	*sync.RWMutex
```

Comment:
If you don't want to use `Once`, better to add mutex per specific field, not for the entire config